### DISCUSSION TITLE
First Hours on GitHub

### DISCUSSION THREAD

**Discussion by JamesCodingSpace** [JamesCodingSpace/Main#1] [General]
Today we set up our brand new github accounts in class.
As a german I might start writing in german soon.
Depends on my motivation.
Whatever, we created a repository and a coding space.
We might start coding as well in the near future.

**Comment by ActiveClientMods** [comment]
Hi m8, im a fellow german too.  It was a bit much at the start, but i think things work out at the end!

**Comment by SaskiaStier** [comment]
Hi !
I hope we start coding in near future too.
You might write in german , I thought I should try writing in english. But same as you it will depend on my motivation. ;)

**Comment by FlipsG** [comment]
Stay focused you got this.
